Pipfile

```diff
@@ -8,6 +8,7 @@ pylint = "*"
 mypy = "*"
 cx-freeze = ">=6.2"
 autopep8 = "*"
+pyqt5 = "*"
```

Comment:
Does PySide not ship with the translation scripts? I'm hesitant depending on two separate Qt distributions, even if only in the dev dependencies.

Reply:
Yes. PySide provides `pyside2-lupdate`. However, it can't recognize the `TRANSLATE` function we used to generate the translation. It seems `pyside2-lupdate` [doesn't support](https://manpages.debian.org/experimental/pyside2-tools/pyside2-lupdate.1.en.html) the custom function name like `pylupdate5`. I'll dig around to find there is any solution.

Reply:
I'll change the uppercase `TRANSLATE` to lowercase `translate`, and remove the pyqt5.